src/librustdoc/clean/types.rs

```diff
@@ -664,7 +659,8 @@ impl Item {
     }
 
     crate fn is_fake(&self) -> bool {
-        matches!(self.def_id, FakeDefId::Fake(_, _))
+        // FIXME: Find a better way to handle this
+        !matches!(self.def_id, ItemId::DefId(..))
```

Comment:
This seems fine to me.

Reply:
Actually I kind of think we should remove this altogether and call !is_def_id instead.